lib/etas/overlap_table_update.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "services"
+require "utils/waypoint"
+require "date"
+require "etas/cluster_update"
+
+module ETAS
+
+  # An update of the table used for ETAS: holdings_htitem_htmember
+  class OverlapTableUpdate
+    attr_accessor :cutoff_date, :waypoint, :num_deletes, :num_adds
+
+    def initialize(cutoff_date = nil)
+      @cutoff_date = cutoff_date || Date.today - 1.5
+      @waypoint = Utils::Waypoint.new(BATCH_SIZE)
+      @num_deletes = 0
+      @num_adds = 0
+    end
+
+    def keep_alive(session, seconds = 120)
+      Thread.new do
+        loop do
+          sleep(seconds)
+          session.client.command(refreshSessions: [session.session_id])
+        end
+      end
+    end
+
+    def overlap_table
+      Services.holdings_db[TABLENAME]
```

Comment:
I would rather see (in `services.rb`):

```ruby
Services.register(:holdings_db_overlap_table) { Services.holdings_db[:holdings_htitem_htmember] }
```

or something along those lines, and getting rid of the `stub_const` calls

Reply:
I registered it as :relational_overlap_table. I am starting to think references to 'holdings_db' are a problem that needs addressing and don't want to perpetuate it.